canonical/store-components
# add the total number of reviews next to ratings

## Done
added the total number of reviews next to star ratings for card component

## QA
- Go to https://store-components-33.demos.haus/?path=/story/packagecard--ratings
- See if there is the total number of reviews next to star ratings

## Fixes
Fixes: https://warthogs.atlassian.net/browse/WD-2584

## Screenshot
<img width="1185" alt="Screen Shot 2023-03-17 at 3 41 43 PM" src="https://user-images.githubusercontent.com/90341644/225953142-a2c4f7c2-49d2-4271-8457-e9cbcc71529e.png">
